src/gui/preferences.c

```diff
@@ -622,7 +622,7 @@ static void init_tab_accels(GtkWidget *book)
   hbox = gtk_box_new(GTK_ORIENTATION_HORIZONTAL, 5);
 
   // Adding the restore defaults button
-  button = gtk_button_new_with_label(C_("preferences", "default"));
+  button = gtk_button_new_with_label(C_("preferences, button", "default"));
```

Comment:
> C_("preferences, button", "default")

Maybe just button?

C_("button", "default")

We really want to find the simplest context and mre generic one. I don't thins a "preference button" would be handled differently than a "button" anywhere in the interface.

Reply:
Agreed; now changed :)